yacl/utils/circuit_executor.cc

```diff
@@ -32,22 +32,44 @@ void PlainExecutor<T>::LoadCircuitFile(const std::string& path) {
   circ_ = reader.StealCirc();
 }
 
+std::string dynamic_bitset_to_string(const dynamic_bitset<>& bits) {
+  std::string str(bits.size(), '0');  // 创建一个与位集大小相同的字符串
+  for (size_t i = 0; i < bits.size(); ++i) {
+    if (bits[i]) {
+      str[bits.size() - 1 - i] = '1';  // 反向填充字符串
+    }
+  }
+  return str;
+}
+
 template <typename T>
-void PlainExecutor<T>::SetupInputs(absl::Span<T> inputs) {
-  YACL_ENFORCE(inputs.size() == circ_->niv);
+void PlainExecutor<T>::SetupInputs(
```

Comment:
You should re-design a new SetupInputs method, maybe just accept a pointer, along with a counter, then deal with the messy thing yourself.

```c++
template <typename T>
void PlainExecutor<T>::SetupInputs(T* inputs, uint64_t len); 
```

Reply:
Update: Can ref to sz's PR: https://github.com/Jamie-Cui/yacl-r/pull/3/files#diff-f4687cb1b2f800a735825a0b6a0a37fbfccf2a4b43e62b4ce484eb4c950bb419